ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -41,38 +41,26 @@ interface StateProps {
   org: Organization
 }
 
-interface DispatchProps {
-  setCommunityTemplateToInstall: typeof setCommunityTemplateToInstall
-}
-
-type Props = DispatchProps &
-  StateProps &
-  RouteComponentProps<{templateName: string}>
-
-// works specifically for csgo, the greatest community template ever conceived
-// https://github.com/influxdata/community-templates/tree/master/csgo
-const getRawYamlFromGithub = repoUrl => {
-  return repoUrl
-    .replace('github.com', 'raw.githubusercontent.com')
-    .replace('tree/', '')
-}
+type Props = StateProps & RouteComponentProps<{templateName: string}>
 
 @ErrorHandling
 class UnconnectedTemplatesIndex extends Component<Props> {
   state = {
-    currentTemplate: '',
+    templateUrl: '',
   }
 
   public componentDidMount() {
-    const {templateName} = this.props.match.params
-    if (templateName) {
-      const currentTemplate = getGithubUrlFromTemplateName(templateName)
-
-      this.setState({currentTemplate}, () => {
-        this.reviewTemplateResources(
-          this.props.org.id,
-          getTemplateNameFromGithubUrl(currentTemplate)
-        )
+    const match = matchPath(this.props.location.pathname, {
+      path: communityTemplatesImportPath,
+    }) as {
+      params: {
+        templateName: string
+      }
+    }
+
+    if (match && match.params && match.params.templateName) {
```

Comment:
```suggestion
    if (match?.params?.templateName) {
```
We can now use the new chaining operator hotness (if you'd like)

Reply:
sweeeeeeeet! make the machines do the work for me.